samples/trumpify/trumpify.py

```diff
@@ -0,0 +1,87 @@
+import os
+import sys
+import threading
+import time
+
+sys.path.append(os.path.abspath('../../'))
+
+from samples.trumpify.utils import TwitterUtils
+from daemo.client import DaemoClient
+
+CREDENTIALS_FILE = 'credentials.json'
+
+PROJECT_ID = -1
+
+INPUT_TWITTER_NAME = 'HillaryClinton'
+MINUTES = 60
+FETCH_INTERVAL_MIN = 5
+TWEET_COUNT = 10
+MONITOR_INTERVAL_MIN = 60
+
+twitter = TwitterUtils()
+client = DaemoClient(CREDENTIALS_FILE)
+
+
+def transform_new_tweets(count, interval):
+    while True:
+        messages = twitter.fetch_tweets(twitter_name=INPUT_TWITTER_NAME, count=count, interval=interval)
+
+        if len(messages) > 0:
+            for message in messages:
+                translate_to_trump_version(message)
+        else:
+            print "@%s has not tweeted in the last %d minutes." % (INPUT_TWITTER_NAME, interval)
+
+        time.sleep(interval)
+
+
+def translate_to_trump_version(message):
+    text = message.get('text')
+    id = message.get('id')
+
+    client.publish(project_id=PROJECT_ID, tasks=[{
+        "id": id,
+        "tweet": text
+    }], approve=approve_tweet, completed=post_to_twitter)
+
+
+def get_tweet_text(worker_response):
+    return worker_response.get('results')[0].get('result')
+
+
+def approve_tweet(worker_responses):
+    approvals = []
```

Comment:
can we do this via a list comprehension to make it simpler? something like (haven't debugged:)
`approvals = [len(get_tweet_text(response)) > 0 for response in responses]`